### DISCUSSION TITLE
Scraping the Recommendation Rating dates

### DISCUSSION THREAD

**Discussion by tamireiny** [ranaroussi/yfinance#1767] [Ideas]
Hello, great toolbox!
I wanted to raise need to get the dates analysts set price target and recommendation, currently the recommendation mean is available via the API, but might be very outdated at certain cases, by adding **dates** of the latest upgrades or downgrade it will be like confidence measure for the mean value. 

ticker_info = yf.Ticker(ticker)
ticker_info_data = ticker_info.info
yahoo_rating = ticker_info_data['recommendationMean']

Attached images to help demonstrate:  
![image](https://github.com/ranaroussi/yfinance/assets/37779442/6c042083-01c7-4f8b-aaf8-1414d5108256)

Thanks for your attention :)

[a comment by an automated account is not shown]

**Reply by tamireiny** [reply]
@bot-unit Thanks for quick response. Can you please add a code snippet that show how to reach the table you mentioned?

**Reply by ValueRaider** [reply]
@bot-unit Release 0.3 can wait

Edit: nonsense? 0.3 should be independent from dev -> main -> release 

Edit2: I think I get question. Ideally feature stays branch until get some feedback from eager users @tamireiny

[a comment by an automated account is not shown]

**Comment by tamireiny** [comment]
Thanks @bot-unit , I am using the latest release,  so I understand that it's not available the moment. 
Any idea when this part will be merged to main?

**Reply by ValueRaider** [reply]
Read #1080

**Reply by tamireiny** [reply]
@bot-unit Works well, you are awesome! 👑
Do we know how if yahoo finance takes recommendation date into account in the recommendation mean? or it's simply: 1 strong buy, 2 buy, 3 hold, 4 sell and 5 strong sell ?

[a comment by an automated account is not shown]

**Comment by tamireiny** [comment]
@bot-unit Thanks for update!
